nntrainer/compiler/tflite_opnode.h

```diff
@@ -93,6 +94,12 @@ class TfOpNode {
    */
   void setNeedReorderWeight() { need_reorder_weight = true; }
 
+  /**
+   * @brief Set the Trainable object
+   *
+   */
+  void setTrainable() { is_trainable = true; }
```

Comment:
you might needs
``` c++
void setTrainable (bool trainable){ is_trainable = trainable;}
```

so that you can set ```is_trainable=false;```

Reply:
[Fix] Thank you for Review